src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/AccountingXmlDocumentEntryFixture.java

```diff
@@ -459,6 +459,37 @@
                     AccountingXmlDocumentBackupLinkFixture.CORNELL_INDEX_PAGE,
                     AccountingXmlDocumentBackupLinkFixture.DFA_INDEX_PAGE),
             CuDisbursementVoucherDocumentFixture.JANE_DOE_DV_DETAIL),
+    DV_DOC_TEST1(1, CuFPTestConstants.DISBURSEMENT_VOUCHER_DOC_TYPE,

```

Comment:
The new fixtures were placed right in between some other ones that go together for a separate test case. Can this new fixture be moved to a different position, to make it clear that it should not be grouped with the "MULTI_DOC_TYPE_TEST..." fixtures?

Reply:
Yes, I shall move them down